### DISCUSSION TITLE
Welcome to Thyme Discussions!

### DISCUSSION THREAD

**Discussion by OmniBlade** [TheAssemblyArmada/Thyme#242] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xezon** [comment]
Hello.

**Reply by z19a** [reply]
I spend a whole nigh trying to optain the exe and dll to test it.
Do you have them complied ready?
